### PR TITLE
more fixes :)

### PR DESCRIPTION
**Pull request by MorganHanson18**

### CODE REVIEW

File: src/app/details/details.component.ts [hunk 1]
```diff
@@ -14,19 +15,13 @@ import { RouterModule } from '@angular/router';
 
 export class DetailsComponent {
   route: ActivatedRoute = inject(ActivatedRoute);
-  todoTitle?: string
-  todoUser?: number
-  todoId?: number
-  todoCompleted?: boolean;
+  todo?: Todo;
```
Comment (lorenzodianni) [inline src/app/details/details.component.ts:18]: yes! this is exactly the improvements that i had in mind! 🚀

File: src/app/details/details.component.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <div>
-    <h1>{{todoId}}.</h1>
-    <h2>{{todoTitle}}</h2>
-    <h3>completed: {{todoCompleted}}</h3>
-    <p>User: {{todoUser}}</p>
+    <h1>{{todo?.id}}.</h1>
+    <h2>{{todo?.title}}</h2>
+    <h3>completed: {{todo?.completed}}</h3>
+    <p>User: {{todo?.userId}}</p>
     <a [routerLink]="['']">back</a>
```
Comment (lorenzodianni) [inline src/app/details/details.component.html:6]: this link works as expected?

Comment (MorganHanson18) [reply src/app/details/details.component.html:6]: yep :)

**Comment by lorenzodianni** [comment]
super when you want merge it in main branch ;)

**Pull request merged**